mathics/builtin/arithmetic.py

```diff
@@ -82,7 +82,7 @@
 )
 
 
-@lru_cache(maxsize=1024)
+@lru_cache(maxsize=4096)
```

Comment:
Why this number?

Reply:
Experiments show that we are exceeding 1024.